download/index.html

```diff
@@ -51,6 +51,7 @@
             <li><a href="https://packages.gentoo.org/package/dev-lang/mono">Gentoo</a></li>
             <li><a href="https://software.opensuse.org/package/mono-complete">openSUSE</a></li>
             <li><a href="http://packages.ubuntu.com/mono-complete">Ubuntu</a></li>
+            <li><a href="http://sotirov-bg.net/slackpack/search.cgi?q&equals;mono">Slackware</a></li>
```

Comment:
This section is for packages in the official package repositories of the various distributions, could you please move it down to "Community packages"?

Also no need to escape the = as `search.cgi?q=mono` will work fine (the current way doesn't work).